### DISCUSSION TITLE
nav_min_ground_speed

### DISCUSSION THREAD

**Discussion by petrweigl** [iNavFlight/inav#9887] [Q&A] [answered]
I am sorry if  I miss something, "Minimum ground speed for navigation flight modes [m/s]. Default 7 m/s. Values: 6 - 50 Default: 7" is this also for MC? if I need just go waypoints mission 2m/s, is that possible, right?

**Comment by Jetrell** [comment, marked as answer]
That value is for fixedwing use. Because they don't have a throttle based speed control setting.. That value just ensuring the plane makes headway into a strong wind.
While multicopter navigation speed is set according to your requirements below. 

![1](https://github.com/iNavFlight/inav/assets/47995726/6620c727-d307-4222-83fa-0ba1ae1c14dc)

By adjusting _default navigation speed_ and _max navigation speed_. But you may also need to reduce _max. bank angle_ a little, to ensure it doesn't runaway, and keeps the 7.2km/h you require.

And if you wish. You could also set maximum speed limits for each waypoint in the mission control.

**Comment by MrD-RC** [comment]
Yes, currently fixed wing only. I'll update Settings.md to state that.
